renderer/chrome.go

```diff
@@ -63,18 +63,16 @@ func listenForResponse(c chan *network.ResponseReceivedReply, responseReceivedCl
 
 func CreatePdf(ctx context.Context, params ChromeParameters) ([]byte, []byte, error) {
 	// Use the DevTools API to manage targets
-	devt := devtool.New("http://127.0.0.1:9222")
-	pt, err := devt.Get(ctx, devtool.Page)
+	devt, err := devtool.New("http://127.0.0.1:9222").Version(ctx)
```

Comment:
I'm not familiar with what Version is doing. What's the reasoning behind this change?

Reply:
I was getting an error `cdp.Target: CreateBrowserContext: rpc error: Not allowed. (code = -32000)` when attempting to render a page. I found this issue (https://github.com/mafredri/cdp/issues/80) where the author suggested using `.Version` which seemed to have fixed my issue.